www/src/lore/Shun_He.md

```diff
@@ -4,6 +4,8 @@ path: /wiki/Shun_He
 layout: wiki
 ---
 
+## Shun He
```

Comment:
![localhost_8000_wiki_Shun_He(iPhone X) (1)](https://user-images.githubusercontent.com/2229300/86048410-56145f80-ba05-11ea-8952-2dc6ffc5f583.png)
![Uploading localhost_8000_wiki_Shun_He(iPhone X).png…]()



probably want to revisit the sizing of images on mobile, sometimes it looks right, sometimes not, really have to take control of floating